ProjectSetup/Repositories/PostRepository.cs

```diff
@@ -50,12 +49,9 @@ public async Task<Post> CreatePost(Post post)
 			return post;
 		}
 
-		public async Task<Post> UpdatePost(UpdatePostRequest postRequest)
+		public async Task<Post> UpdatePost(Post postRequest)
 		{
-			Post post = await FindByCondition(post => post.Id.Equals(postRequest.Id))
```

Comment:
Why did you remove this?

Reply:
I removed this because I'm getting post by id in PostController